_projects/2024/100446517/100446517.Rmd

```diff
@@ -0,0 +1,692 @@
+---
+title: "Book reading habits"
+description: |
+  A short description of the post.
+categories: "2024"
+author: Aurora Sterpellone
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+Distill is a publication format for scientific and technical writing, native to the web.
+
+Learn more about using Distill at <https://rstudio.github.io/distill>.
+
+# EU Book Reading Habits Graph Replication
+
+## Introduction
+
+In 2022, according to EU statistics on income and living conditions, 52.8% of the EU population aged 16 years or over reported reading books in the past 12 months.
+
+The “Book Reading Habits over the Past 12 Months (2022)” graph, created by Eurostat, the statistical office of the European Union, provides an insightful look into reading habits across different EU countries and a few non-EU countries.
+
+The data reflects not only cultural and educational trends in book readership but also highlights variations between countries, offering a window into broader societal behaviors. This information is valuable for understanding literacy levels, cultural engagement, and potential market demand in the publishing industry across Europe and its neighboring regions.
+
+![](images/books-day-MS-2022.png)
+
+## Necessary Libraries
+
+The following libraries are used for the graph replication and the graph improvement.
+
+```{r}
+#| echo: true
+#| results: hide
+
+library(ggplot2)
+library(dplyr)
+library(tidyr)
+
+library(sysfonts)
+library(showtext)
+```
+
+## Load the data
+
+Load the dataset into the environment. Here, ilc_scp27_linear represents the dataset containing information on book reading habits.
+
+```{r}
+data <- read.csv(file = "ilc_scp27_linear.csv")
+```
+
+## Data cleaning and transformation
+
+### **Step 1: Filter Relevant Data**
```

Comment:
Please remove any additional formatting like this from all the titles to ensure a uniform style with other articles. I'm referring to the bold text.